daily-challenges/2024-11/10/ththao27.spec.ts

```
@@ -0,0 +1,46 @@
+function analyzeSteps (dailySteps: number[], target: number = 10000): StepsAnalysis {
+    const totalSteps = dailySteps.reduce((sum, currentValue) => sum + currentValue, 0);
+    const averageSteps = Math.floor(totalSteps / dailySteps.length);
+    const bestDay = Math.max(...dailySteps);
+    const worstDay = Math.min(...dailySteps);
+    const daysAboveTarget = dailySteps.filter((day) => { 
+        return day >= target;
+    }).length;
+    const countStreak = (daylySteps: number[]): number => {
+        let currentStreak: number = 0;
+        let maxStreak: number = 0;
+        daylySteps.forEach((day) => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)